### PR TITLE
Remove all articles from blacklist

### PR DESCRIPTION
**Pull request by tomas-stefano**
* There are some harded coded pages that are blocked from preview.
Please, let's remove that.

### CODE REVIEW

**Comment by esbel** [comment]
:+1:

**Comment by dougdroper** [comment]
As discussed, we need to be careful that we are not hiding the intention of the page :)

**Comment by tomas-stefano** [comment]
Yes, I send to the business this blacklisted pages. I will wait and see for preview and published states.

**Comment by tomas-stefano** [comment]
@dougdroper we decide to remove all articles from blacklist and test on QA all this pages. Can you review now?

File: config/routes.rb [hunk 1]
```diff
@@ -111,8 +111,7 @@ def not_implemented
               end
 
     get '/:page_type/:id/preview' => 'articles_preview#show',
-        page_type: /articles|action_plans|news|videos|corporate|tools/,
-        constraints: ValidResource.new(:article)
```
Comment (dougdroper) [inline config/routes.rb]: do we need to remove the constraints from the article route?

Comment (tomas-stefano) [inline config/routes.rb]: There is no articles anymore. The call will make anything.

Comment (tomas-stefano) [inline config/routes.rb]: There are move valid resource call. I will remove all blacklisted articles Valid resource that is being called.

Comment (tomas-stefano) [inline config/routes.rb]: Removed!!!!!

File: config/initializers/valid_resource.rb [hunk 2]
```diff
@@ -1,20 +1,9 @@
 class ValidResource
   BLACKLIST = {
-    article:     %w(about-our-debt-work am-ein-gwaith-dyled
-                    debt-publications cyhoeddiadau-ar-ddyledion
-                    partners-overview-parhub
```
Comment (dougdroper) [inline config/initializers/valid_resource.rb]: cool, there are some redirects in scripts, that we may need to double check that they are still needed, but :+1:

**Comment by dougdroper** [comment]
:+1:

**Pull request merged**